.dockerignore

```diff
@@ -0,0 +1,16 @@
+#exclude heaviest object in the universe
+node_modules
+
+#exclude e2e tests
+test
+
+#exclude unit tests
+**/*.spec.ts
+
+#no need to copy 'dist' directory into docker image
+dist
+
+#no need to copy these files into docker image
+Dockerfile
+README.md
+
```

Comment:
Just a note that does not affect the final mark. you can also exclude git related files like `.git` folder